proto/spacemesh/v1/smesher_types.proto

```diff
@@ -127,4 +127,21 @@ message PostSetupStatus {
     uint64 num_labels_written = 2; // Number of labels (hashes) written to the data files
     PostSetupOpts opts = 3; // setup options previously set by the user
     string error_message = 4; // The error message, if the state is STATE_ERROR
+    string processed = 5; // The processed size, like `176 GB`
```

Comment:
I don't think this should be string, this should be a number, probably number of bytes